roborock/api.py

```diff
@@ -293,10 +293,7 @@ def on_message_received(self, messages: list[RoborockMessage]) -> None:
                     payload = data.payload[0:24]
                     [endpoint, _, request_id, _] = struct.unpack("<8s8sH6s", payload)
                     if endpoint.decode().startswith(self._endpoint):
-                        try:
-                            decrypted = Utils.decrypt_cbc(data.payload[24:], self._nonce)
-                        except ValueError as err:
-                            raise RoborockException("Failed to decode %s for %s", data.payload, data.protocol) from err
+                        decrypted = Utils.decrypt_cbc(data.payload[24:], self._nonce)
```

Comment:
Why raise ValueError?

Reply:
Although 0 i'm not sure why i deleted it. thta was a mistake i will add back